packages/changed-elements-react/src/widgets/V2Widget/models/ComparisonJobModels.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * Job status used for identification of job progress
+ * Unknown Job Status is reserved for when a job has not yet been queried.
+ * Only use unknown when no job status is available.
+*/
+export type JobStatus = "Unknown" | "Available" | "Not Processed" | "Processing" | "Error" | "Queued";
```

Comment:
What's the difference between `"Not Processed"` and `"Queued"`?

Reply:
`Not Processed` means it hasn't even been requested to be processed, while `Queued` means the processing has started but the agent's haven't started.

Reply:
Thanks, this is very useful information. Consider adding this to the documentation.